### PR TITLE
[js][bidi] add Network module events

### PR DESCRIPTION
**Pull request by TamsilAmani**
### Description

Add network module events and types

### Motivation and Context

Implement network module events and types as described in https://w3c.github.io/webdriver-bidi/#module-network

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: javascript/node/selenium-webdriver/bidi/networkInspector.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    await this.subscribeAndHandleEvent('network.responseCompleted', callback)
+  }
+
+  async subscribeAndHandleEvent(eventType, callback) {
+    if (this._browsingContextIds != null) {
+      await this.bidi.subscribe(eventType, this._browsingContextIds)
+    } else {
+      await this.bidi.subscribe(eventType)
+    }
+    this._on(callback)
```
Comment (harsha509) [inline javascript/node/selenium-webdriver/bidi/networkInspector.js]: missing await!

Comment (TamsilAmani) [reply javascript/node/selenium-webdriver/bidi/networkInspector.js]: Fixed

**Review by harsha509 — APPROVED** [review]

**Comment by StanislavKharchenko** [comment]
Glad to see this functionality coming soon for JS binding.
Could you please also make some relevant examples for this part?
It will be great to know how to see all pending and/or completed http requests, which application send (as we can see it in the browser "Network" tab).

**Comment by TamsilAmani** [comment]
Hey @StanislavKharchenko. The documentation will be updated soon for the BiDi modules. Meanwhile, I have added tests to this PR also. You can have a look at them to get some idea. Though they aren't written from a user perspective but more from a testing point of view.

**Pull request merged**